core/templates/pages/topic-editor-page/questions-tab/topic-questions-tab.directive.html

```diff
@@ -1,7 +1,7 @@
 <md-card class="oppia-page-card oppia-long-text" ng-if="!questionEditorIsShown && !activeQuestion">
   <div class="questions">
     <h3 class="form-heading"> Questions included in this topic linked to selected skill </h3>
-    <select class="form-control protractor-test-select-skill-dropdown"
+    <select class="form-control select-css protractor-test-select-skill-dropdown"
```

Comment:
This class name is too generic. Can we call it skill-select-dropdown or similar?

Reply:
Done.